tokenizers/src/pre_tokenizers/metaspace.rs

```diff
@@ -9,10 +9,20 @@ use crate::tokenizer::{Decoder, PreTokenizedString, PreTokenizer, Result, SplitD
 pub struct Metaspace {
     replacement: char,
     pub add_prefix_space: bool,
+    #[serde(skip_serializing_if = "skip_legacy_serialization")]
+    pub legacy: bool,
```

Comment:
for consistency, let's try to properly always save it, gotta check FC.

Reply:
rename "only_first" or "prepend_once" etc

Reply:
ENUM prepend, once, never to support statefull-like changes to this. More descriptive